### ISSUE TITLE
Document ccitt limitations for tiff

### ISSUE THREAD

**Issue by mschneider82** [hhrutter/pdfcpu#63]
Just tried to replace golang.org/x/image/tiff with yours, but
when i try tiff encoding with compression setting CCITTGroup3 or CCITTGroup4 i will get:

> main.go:39
>  err = tiff.Encode(f, img, &tiff.Options{Compression: tiff.CCITTGroup4})

```
goroutine 1 [running]:
github.com/hhrutter/pdfcpu/tiff.writePix(0x0, 0x0, 0xc0000b8000, 0x21335d0, 0x21335d0, 0xdb4, 0x26c4, 0x26c4, 0xc000090078, 0xc002204bf8)
        /home/ms/go/src/github.com/hhrutter/pdfcpu/tiff/writer.go:240 +0x4f
github.com/hhrutter/pdfcpu/tiff.encodeRGBA(0x0, 0x0, 0xc0000b8000, 0x21335d0, 0x21335d0, 0x9b1, 0xdb4, 0x26c4, 0x7f9cf7b93100, 0x5, ...)
        /home/ms/go/src/github.com/hhrutter/pdfcpu/tiff/writer.go:110 +0x319
github.com/hhrutter/pdfcpu/tiff.Encode(0x757880, 0xc000090078, 0x757f80, 0xc002208000, 0xc002204eb0, 0xc0021ec040, 0xb)
        /home/ms/go/src/github.com/hhrutter/pdfcpu/tiff/writer.go:423 +0x11ae
main.main()
        /home/ms/go/src/pdftest/main.go:39 +0x2a5

```

**Comment by hhrutter** [comment]
CCITT Encoding is not supported yet.
There is only CCITT Decoding for the most popular variants.

I have to document this and catch this in pdfcpu's TIFF package.
Thanks for reporting this!

**Comment by hhrutter** [comment]
Fixed with [v0.1.23](https://github.com/hhrutter/pdfcpu/releases/tag/v0.1.23)
